impl-endpoints/nss/Dockerfile

```diff
@@ -22,8 +22,8 @@ RUN cd /build \
    && hg clone https://hg.mozilla.org/projects/nspr \
         --rev b09175587dad2bfb923ec87250ac80461f620577 \
    && hg clone https://hg.mozilla.org/projects/nss \
-        --rev 38a91427d65fffd0d7f7d2b6d0bcee7dc8b77a37 \
    && cd nss \
+   && hg pull -u -r 98542d9c204f8e91336f0a36239d776d82dc8989 https://hg.mozilla.org/projects/nss-try \
```

Comment:
string and cellotape!

Reply:
These change will be reverted once your PR lands.